src/components/Match/MatchContainer.js

```diff
@@ -153,6 +144,36 @@ const MatchContainer = ( ) => {
     onlineSuggestionsAttempted
   } );
 
+  const getCurrentUserLocation = async ( ) => {
+    const currentUserLocation = await fetchUserLocation( );
+    const placeName
+     = await fetchPlaceName( currentUserLocation.latitude, currentUserLocation.longitude );
```

Comment:
There were some user bugs reported with updating this immediately as done here. If you compare the way it is done in ObsEdit: https://github.com/inaturalist/iNaturalistReactNative/blob/b43350730828b1b43a9a868a06d7008f791c20d1/src/components/ObsEdit/EvidenceSectionContainer.js#L148

Not sure if we need the same safeguards here. But maybe better to follow exactly what we are doing there.